ow_faults_detection/include/ow_faults_detection/FaultDetector.h

```diff
@@ -13,6 +13,7 @@
 #include <std_msgs/Float64.h>
 #include <std_msgs/Empty.h>
 #include <ow_faults_detection/JointStatesFlag.h>
+#include <owl_msgs/ActionGoalStatus.h>
```

Comment:
I think this should be moved to `ow_lander` for now, to remain consistent since it's not a unified topic.